docs/build/reference/dynamic-deopt.md

```diff
@@ -54,8 +55,8 @@ If you specify `/OPT:ICF`, the compiler generates a warning that the debug exper
 /fastcap
 /callcap
 /ZW
-fsanitize=address
-fsanitize=kernel-address
+/fsanitize=address
+/fsanitize=kernel-address
 All of the CLR flags
 ```
```

Comment:
The code block above could be converted to a list of links such that the reader can go to the respective reference pages if they are unsure what the switch does.